DataFormats/simulation/include/SimulationDataFormat/Stack.h

```diff
@@ -145,6 +145,10 @@ class Stack : public FairGenericStack
 
     virtual void Print(Int_t iVerbose = 0) const;
 
+    /// Output to screen (derived from base class)
+    /// \param option: 0=events summary, non0=track info
+    virtual void Print(Option_t* option = 0) const;
+
```

Comment:
This is even in the coding guidelines, see
https://htmlpreview.github.io/?https://github.com/AliceO2Group/CodingGuidelines/blob/master/coding_guidelines.html#Inheritance